docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx

```diff
@@ -174,6 +174,56 @@ d. Top cap in white is the rest of M5 Miner.
 </div>
 </details>
 
+
+## Other Issues
+
+### M5 not working in 5 GHz WiFi.
+M5 is designed to work **only** at the 2.4GHz frequency. Please make sure the 2.4Ghz band on your router is enabled. If you are still experiencing issues, please contact us through [support](https://support.weatherxm.com)
+
+<hr />
+
+### My M5 is stuck/unresponsive.
+  
+Please do not restart it and contact us through [support](https://support.weatherxm.com) to debug it and find the reason of the issue
+
+<hr />
+
+### The GPS sign on my M5 is red.
+
+Please check that the plug is firmly placed, then move the M5 with the GPS antenna outside for some minutes and place it close to a window. 
+
+<hr />
+
+
+### Pressure on M5 appears to be very low relatively to other reference stations of national or private networks etc. 
+
+For now we present only the absolute pressure and not the Mean Sea Level Pressure (MSLP). That means that the displayed pressure are affected by the elevation of your area. We work to consider the elevation given by your GPS in order to show the MSLP, which is commonly used.
+
+<hr />
```

Comment:
Let's move it above under the "Weather Station" section as more weather-variables-related issues are there.